config/crd/bases/test.openstack.org_tobikoes.yaml

```diff
@@ -93,6 +99,47 @@ spec:
                 default: py3
                 description: Test environment
                 type: string
+              toleration:
```

Comment:
should it be `toleration` or `tolerations`?
Is it important?

Reply:
Yes, I think it is important. Let me fix it:). The change would probably work but the name of the parameter would be inconsistent with the tempest one.